### PR TITLE
CGSN config page

### PR DESCRIPTION
**Pull request by c0mster**
Please test and advise of any issues. Developed on windows so there may be a bit of a tweak for the config file having, or not having \r on Linux. Developed with Linux in mind so \r should not be relevant.

Thanks

### CODE REVIEW

**Comment by birdage** [comment]
@DanielJMaher i think this is on you buddy

**Comment by DanielJMaher** [comment]
@birdage I can review for how it works, can you make sure the coding is up to snuff!! no offense @c0mster lol.

File: ooiui/static/js/models/common/CGSNConfigModel.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            ,{lines:[282,565],valid:function(v){return (this.pattern.exec(v)==null?false:true)},pattern:new RegExp("^[0-5]$"),sample:"0 - 5 "}
+            ,{lines:[252,258,265],valid:function(v){return (this.pattern.exec(v)==null?false:true)}
+                ,pattern:new RegExp("^$|^[0-9]{4}(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])$")
+                ,sample:"20120101"}
+            ,{lines:[159],valid:function(v){return (this.pattern.exec(v)==null?false:true)}
+                ,pattern:new RegExp("^200$|^300$")
+                ,sample:"200 or 300"}
+            ,{lines:[226,227,228,229],valid:function(v){return (this.pattern.exec(v)==null?false:true)}
+                ,pattern:new RegExp("^[0-1]:([0-9]|[1-1][0-9]|[2-2][0-3])(,(([0-9]|[1-1][0-9]|[2-2][0-3]),([0-9]|[1-1][0-9]|[2-2][0-3])))*(,([0-9]|[1-1][0-9]|[2-2][0-3]))*:([0-9]|[0-5][0-9]|60)*:([0-9]|[0-5][0-9]|60)$")
+                ,sample:"1:0,4,8,12,16,20:10:20"}
```
Comment (birdage) [inline ooiui/static/js/models/common/CGSNConfigModel.js]: this is one way of doing it!

**Comment by birdage** [comment]
@DanielJMaher this just needs to work, and get added, not spending to much time on it

**Pull request merged**